packages/rulesets/src/spectral/functions/xms-pageable-for-list-calls.ts

```diff
@@ -1,9 +1,15 @@
+import { isNull } from "lodash"
+
 const xmsPageableForListCalls = (swaggerObj: any, _opts: any, paths: any) => {
   if (swaggerObj === null) {
     return []
   }
   const path = paths.path || []
-
+  if (!isNull(path[1])) {
```

Comment:
can you add a line describing what is path[1] and why we verify the RG scope upon non-null